platforms/forge/src/main/java/com/pg85/otg/forge/commands/OTGCommandHandler.java

```diff
@@ -0,0 +1,138 @@
+package com.pg85.otg.forge.commands;
```

Comment:
Very nice refactoring of Forge commands, also great that Forge<>Spigot are more aligned now!